test/integration/targets/mongodb_replicaset/aliases

```diff
@@ -3,4 +3,5 @@ shippable/posix/group1
 skip/osx
 skip/freebsd
 skip/rhel
+linux/centos8/1
```

Comment:
Why the new alias? We don't have any others similar to this one.

Reply:
I don't recall this. Removed it.